message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1556,5 +1556,12 @@
       <field name="AOA" type="float" units="deg">Angle of Attack (degrees)</field>
       <field name="SSA" type="float" units="deg">Side Slip Angle (degrees)</field>
     </message>
+    <!-- wheel traveled distance message -->
+    <message id="11030" name="WHEEL_DISTANCE">
+      <description>Cumulative distances traveled by each wheel in meters (forward rotations increase these values, backward - decrease).</description>
+      <field type="uint64_t" name="time_usec" units="us">Timestamp (microseconds, synced to UNIX time or since system boot)</field>
+      <field type="uint8_t" name="count">Number of wheels reported</field>
+      <field type="float[16]" name="distance" units="m">Distance traveled by each wheel (m)</field>
```

Comment:
remove the "(m)" from the description, it is redundant.

Reply:
Probably the same for microseconds?
Message id 11030 is already used. What number should I use?

Reply:
yes, good catch, the same for microseconds.
Message Id, just use the next free one.

Reply:
Currently, I use 11029 for msg id. Is it ok or better to use 11040?

Reply:
The general advice is that you want to add space in the id range to allow common messages to be grouped. So it depends on whether this message is likely to need to have other related messages "with it" and whether the empty preceding IDs are likely to be needed by the messages before them.
FWIW I would use 11040.